docs/api/protocol.md

```diff
@@ -87,7 +87,7 @@ module gets emitted.
     * `method` String
     * `uploadData` [UploadData[]](structures/upload-data.md)
   * `callback` Function
-    * `filePath` String (optional)
+    * `filePath` ( String | {path:string} | number | {error:any} ) (optional)
```

Comment:
@zeke We can't call it an `Error` object as that is actually a class in typescript.  We could just call it an `ErrorObject`.  Other than that it looks right 👍